fyle_accounting_mappings/models.py

```diff
@@ -314,7 +314,10 @@ class MappingSetting(models.Model):
     """
     id = models.AutoField(primary_key=True)
     source_field = models.CharField(max_length=255, help_text='Source mapping field')
```

Comment:
Can we add unique together constraint for source, destination, workspace? your thoughts?
We anyways have validations at frontend to avoid this, still we update_or_create directly without checking anything in api.

Reply:
Yeah, lets add this too